content/tutorials/single-page-tutorials/mint-nft-with-javascript-shimmer.md

```diff
@@ -0,0 +1,218 @@
+---
+title: How to mint an NFT” with Javascript
```

Comment:
```suggestion
title: How to mint a native NFT with Javascript on Shimmer
```

Same as above.

Reply:
Why did you resolve this?
At least the " has to go 😅